src/pages/404.js

```diff
@@ -0,0 +1,20 @@
+import Link from 'next/link';
+import Footer from '@/components/footer';
+import Layout from '@/components/Layout';
+import styles from '@/components/Layout/Layout.module.scss';
+
+export default function FourOhFour() {
+    return <>
+        <Layout>
+            {/* <div className={styles.layout}> */}
```

Comment:
We don't commit commented code, can we remove this?

Reply:
@NoLayerCode could you please work on this please